xchange-cexio/src/main/java/info/bitrich/xchangestream/cexio/CexioStreamingMarketDataService.java

```diff
@@ -39,17 +39,15 @@ public OrderBookUpdateConsumer(CexioStreamingRawService streamingOrderDataServic
     @Override
     public OrderBook apply(CexioWebSocketOrderBookSubscribeResponse t) throws Exception {
       OrderBook retVal;
-      if (prevID == null || prevID.add(BigInteger.ONE).compareTo(t.id) == 0) {
-        prevID = t.id;
-        orderBookSoFar = CexioAdapters.adaptOrderBookIncremental(orderBookSoFar, t);
-        retVal = orderBookSoFar;
-      } else {
-        streamingOrderDataService.resubscribeChannels();
-        retVal = orderBookSoFar;
-        orderBookSoFar =
+      if (prevID != null && prevID.add(BigInteger.ONE).compareTo(t.id) != 0) {
+        OrderBook orderBookSoFar =
```

Comment:
This line doesn't look like it does anything! Just assigns a local variable which is never used.

Reply:
Lol good catch, that's what I get for copy-pasting the assignment in the declaration. The statement is supposed to assign to the member variable, fixed.